compiler-core/src/parse/snapshots/gleam_core__parse__tests__assignment_pattern_invalid_bit_segment.snap

```diff
@@ -8,6 +8,6 @@ error: Syntax error
 3 │     let <<b1, pub>> = <<24, 3>>
   │               ^^^ I was not expecting this
 
-Expected one of: 
+Found "pub" (a keyword), expected one of: 
```

Comment:
```suggestion
Found the keyword `pub`, expected one of: 
```

Reply:
@lpil Thanks for the suggestion. Backticks are definitely better on the eyes here.

Should this be the case for non-keywords as well, or shall we keep double quotes? I ask because currently the `expected` token that gets printed is also wrapped in double quotes.

Reply:
@lpil Pushed some updates per your feedback and kept the behavior of non-keywords the same. Lmk if that works... thanks!